man/repart.d.xml

```diff
@@ -453,6 +453,10 @@
         exclude multiple files or directories from host from being copied into the newly formatted file
         system.</para>
 
+        <para>Note that when this option is used to exclude a directory, the directory itself is still
+        copied, but none of its children are copied. This is done so that the directory can still serve as a
+        mountpoint if needed.</para>
```

Comment:
hmm, it appears to me that people won't like this. there are likely usecases where you want one behaviour, or when you want the other, you'll always make half the users unhappy.

what about this:

1. if specified path is not suffixed with /: will hide the node altogether, regardless what type it is
2. if specified path is suffixed with "/": will hide the contents of the inode, if it is a directory. if it is not a directory, will not do anything (but probably should warn)

Reply:
(this idea is remotely inspired how in rsync suffixing "/" to a path means to copy contents, not inode of a dir)

Reply:
(you could implement this by simply maintaing two denylist Set objects internally, one that covers inodes, and the other that covers contents of dir inodes)

Reply:
Implemented but via hashmap containing the deny type instead of two denylists